### DISCUSSION TITLE
4.0.7 release?

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#1456] [General]
All devs,

I would like to roll a last 4.0.x fixing release. Here are the included fixes:
https://github.com/sepinf-inc/IPED/blob/4.0.x/ReleaseNotes.txt#L1-L17

Should we include something else?

**Comment by wladimirleite** [comment]
I don't remember anything else that should be added.

**Reply by lfcnassif** [reply]
Thank you @tc-wleite. I'll just include #1454, do basic tests and release.
